lisa/operating_system.py

```diff
@@ -215,53 +215,96 @@ def capture_system_information(self, saved_path: Path) -> None:
     @classmethod
     def _get_detect_string(cls, node: Any) -> Iterable[str]:
         typed_node: Node = node
-        cmd_result = typed_node.execute(cmd="lsb_release -d", no_error_log=True)
-        yield get_matched_str(cmd_result.stdout, cls.__lsb_release_pattern)
+        cmd_result = typed_node.execute(
+            cmd="lsb_release -d", no_error_log=True, timeout=30
```

Comment:
I'm general ok with the change, but the timeout can be set to 60 seconds for some extremely conditions. The key thing is to raise the exception, if first timeout met.

Reply:
Done